eng/common/templates/steps/test-images-linux-client.yml

```diff
@@ -10,9 +10,9 @@ steps:
       optionalTestArgs="-DisableHttpVerification"
     fi
     if [ "${{ eq(variables['System.TeamProject'], 'public') }}" == "False" ]; then
-      optionalTestArgs="$optionalTestArgs -Registry $(acr.server) -RepoPrefix $(stagingRepoPrefix)"
+      optionalTestArgs="$optionalTestArgs -Registry $(acr.server) -RepoPrefix $(stagingRepoPrefix) -TestCategory $(testCategory)"
     else
-      optionalTestArgs="$optionalTestArgs -IsLocalRun"
+      optionalTestArgs="$optionalTestArgs -IsLocalRun -TestCategory $(testCategory)"
```

Comment:
I am concerned about the impact adding the TestCategory parameter has on the common engineering infrastructure.  Do we want to require this parameter on the RunTests script in every repo?  Perhaps it would be better to add support for repo specific test args. By that I mean the `TestCategory` variable you added could be changed to something like `repoTestArgs = "-TestCategory WCF".  Then in this common build step we could append the repoTestArgs to the optionalTestArgs only when the variable is defined.

Reply:
I think I did what yuo want, but I also had to change run-tests.ps1 a little bit. #330

Reply:
Closed #330 and made the changes directly into this branch.